homeassistant/components/smartthings/sensor.py

```diff
@@ -554,6 +554,7 @@ def __init__(
         """Init the class."""
         super().__init__(device)
         self.report_name = report_name
+        self._attr_state_class = STATE_CLASS_MEASUREMENT
```

Comment:
as already discussed here https://github.com/home-assistant/core/pull/53759#discussion_r680026699 and https://github.com/home-assistant/core/pull/53759#discussion_r680630611 the `SmartThingsPowerConsumptionSensor` should not set `STATE_CLASS_MEASUREMENT`

Reply:
we're making an exception because it's close enough.

Reply:
ah ok 👍

Reply:
I am aware, balloob stated this is an exception.